torch/csrc/jit/codegen/cuda/lower_insert_syncs.cpp

```diff
@@ -131,10 +131,6 @@ class WarSyncInserter : private kir::ExprMutator {
   //! Insert Sync nodes at the end of a given for-loop when a WAR
   //! hazard may happen.
   WarSyncInserter(const std::vector<Expr*>& exprs) {
-    auto& lower_alloc_info_map = GpuLower::current()->localAllocationInfoMap();
```

Comment:
This change is not strictly necessary, but we don't need to rely on the `GpuLower` map.